src/components/Composer/index.native.js

```diff
@@ -95,37 +96,24 @@ function Composer({shouldClear, onClear, isDisabled, maxLines, forwardedRef, isC
         onClear();
     }, [shouldClear, onClear]);
 
-    /**
-     * Set maximum number of lines
-     * @return {Number}
-     */
-    const maxNumberOfLines = useMemo(() => {
-        if (isComposerFullSize) {
-            return;
-        }
-        return maxLines;
-    }, [isComposerFullSize, maxLines]);
-
-    const composerStyles = useMemo(() => {
-        StyleSheet.flatten(props.style);
-    }, [props.style]);
+    const maxHeightStyle = useMemo(() => StyleUtils.getComposerMaxHeightStyle(styles, maxLines, isComposerFullSize), [isComposerFullSize, maxLines, styles]);
+    const composerStyle = useMemo(() => StyleSheet.flatten(style), [style]);
 
-    // On native layers we like to have the Text Input not focused so the
-    // user can read new chats without the keyboard in the way of the view.
-    // On Android the selection prop is required on the TextInput but this prop has issues on IOS
-    const propsToPass = _.omit(props, 'selection');
     return (
         <RNTextInput
+            /* eslint-disable-next-line react/jsx-props-no-spreading */
+            {...props}
+            multiline
             autoComplete="off"
             placeholderTextColor={theme.placeholderText}
             ref={setTextInputRef}
             onContentSizeChange={(e) => ComposerUtils.updateNumberOfLines({maxLines, isComposerFullSize, isDisabled, setIsFullComposerAvailable}, e, styles)}
             rejectResponderTermination={false}
-            smartInsertDelete={false}
-            maxNumberOfLines={maxNumberOfLines}
-            style={[composerStyles, styles.verticalAlignMiddle]}
+            smartInsertDelete
```

Comment:
```suggestion
            smartInsertDelete={false}
```

Hey @chrispader 👋 , jumped on this out of curiosity, why are we setting `smartInsertDelete` to `true`? It should be `false` in order to fix [this issue](https://github.com/Expensify/App/pull/15051). At the moment isn't doing anything but it will start working when we upgrade the App to 0.73.